caravel/views.py

```diff
@@ -958,6 +960,58 @@ def explore(self, datasource_type, datasource_id):
                     mimetype="application/json")
             return resp
 
+    @api
+    @has_access_api
+    @expose("/filter/<datasource_type>/<datasource_id>/<column>/")
+    def filter(self, datasource_type, datasource_id, column):
+        """
+        Endpoint to retrieve values for specified column.
+
+        :param datasource_type: Type of datasource e.g. table
+        :param datasource_id: Datasource id
+        :param column: Column name to retrieve values for
+        :return:
+        """
+        # TODO: Cache endpoint by user, datasource and column
```

Comment:
Speaking of cache was wondering if it would make sense to wrap flask-cache's Cache or just create some helpers to avoid spreading the compression and python3 quirks around. What do you think @mistercrunch?